dissect/target/plugins/os/windows/tasks.py

```diff
@@ -351,57 +826,85 @@ def get_actions(self):
 
 
 class TasksPlugin(Plugin):
-    """TODO."""
+    """
+    Plugin for retrieving scheduled tasks on a Windows system.
+    """
 
     PATHS = [
         "sysvol/windows/system32/tasks",
         "sysvol/windows/system32/tasks_migrated",
         "sysvol/windows/syswow64/tasks",
+        "sysvol/windows/tasks",  # at.exe job file location
     ]
 
-    def __init__(self, target):
+    def __init__(self, target: Target):
+        """
+        Initialize the TasksPlugin and build a list with files to parse.
+
+        Args:
+            target: The target system.
+        """
```

Comment:
Same comment as previously, not sure a docstring here is necessary. And if you add one, add it to the class, not `__init__`.